Backend/src/main/resources/application.properties

```diff
@@ -1,12 +1,4 @@
-server.port = 5000
-endpoints.env.enabled=true
-spring.jackson.serialization.FAIL_ON_EMPTY_BEANS=false
-## MySQL
-spring.datasource.username = ${SECRETS_DB_USERNAME}
-spring.datasource.password = ${SECRETS_DB_PASSWORD}
-spring.jpa.hibernate.ddl-auto = update
-spring.jpa.database-platform=org.hibernate.dialect.MySQL5InnoDBDialect
-spring.datasource.url=jdbc:mysql://${SECRETS_DB_URL}
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-jwt.secret= ${SECRETS_JWT}
-sendGrid.key= ${SECRETS_SENDGRID}
\ No newline at end of file
+spring.datasource.username = omegaadmin3@omegadatabase3
+spring.datasource.password = omegaDatabase3
+spring.datasource.url=jdbc:mysql://omegadatabase3.mysql.database.azure.com/omegarestapi?useLegacyDatetimeCode=false&serverTimezone=UTC
+jwt.secret= 'a@a.com'
```

Comment:
This shouldn't have been committed

Reply:
This indeed should have not i excluded it.